src/app/ReadClient.h

```diff
@@ -48,12 +48,12 @@
 #include <messaging/Flags.h>
 #include <protocols/Protocols.h>
 #include <system/SystemPacketBuffer.h>
-
```

Comment:
Why all these blank line removals?  Please don't do that.

Reply:
Done